basiclti-impl/src/java/org/sakaiproject/lti/impl/BaseLTIService.java

```diff
@@ -650,6 +650,11 @@ public Object insertToolContent(String id, String toolId, Properties reqProps, S
 		return insertToolContentDao(id, toolId, reqProps, siteId, isAdmin(siteId), isMaintain(siteId));
 	}
 	
+	public Object insertTIIToolContent(String id, String toolId, Properties reqProps)
```

Comment:
Why not just use a security advisor to pass the permission checks rather than opening up the API with a method that bypasses security check?